app-doc/elisp-manual/elisp-manual-26.3.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
We only update copyright when the ebuilds changed. Did they change this year and someone forgot to update it?

Reply:
I have dropped the commit so now it is restored to original version